src/server/server.test.ts

```diff
@@ -18,91 +23,148 @@ describe('TranscriptREST', () => {
   beforeEach(() => {
     initialize(); // start with a fresh data base
   });
-  afterAll( () => {
+  afterAll(() => {
     server.close();
   });
 
-  describe('createStudent API', () => {
-    test('a new name', async () => {
-      const { data } = await axios.post<{ studentID:number }>(`${baseurl}/transcripts`, { name: 'Ryan' });
+  describe("createStudent API", () => {
+    test("a new name", async () => {
+      const { data } = await axios.post<{ studentID: number }>(
+        `${baseurl}/transcripts`,
+        { name: "Ryan" }
+      );
       expect(data.studentID).toBeGreaterThan(4);
     });
-    test('a reused name', async () => {
-      const { data } = await axios.post<{ studentID:number }>(`${baseurl}/transcripts`, { name: 'avery' });
+    test("a reused name", async () => {
+      const { data } = await axios.post<{ studentID: number }>(
+        `${baseurl}/transcripts`,
+        { name: "avery" }
+      );
       expect(data.studentID).toBeGreaterThan(4);
     });
   });
 
-  describe('getStudent API', () => {
-    test('student present', async () => {
-      const sample : Transcript = randomTranscript();
+  describe("getStudent API", () => {
+    test("student present", async () => {
+      const sample: Transcript = randomTranscript();
       const studentID = addStudent(sample.student.studentName, sample.grades);
       sample.student.studentID = studentID; // update
       const { data } = await axios.get(`${baseurl}/transcripts/${studentID}`);
       expect(data).toStrictEqual(sample);
     });
-    test('student absent', async () => {
+    test("student absent", async () => {
       await expect(axios.get(`${baseurl}/transcripts/6`)).rejects.toThrow();
     });
   });
 
-  describe('getStudentID API', () => {
-    test('not present', async () => {
-      const { data } = await axios.get<number[]>(`${baseurl}/studentids?name=ryan`);
+  describe("getStudentID API", () => {
+    test("not present", async () => {
+      const { data } = await axios.get<number[]>(
+        `${baseurl}/studentids?name=ryan`
+      );
       expect(data).toStrictEqual([]);
     });
-    test('present (new)', async () => {
-      const sample : Transcript = randomTranscript();
+    test("present (new)", async () => {
+      const sample: Transcript = randomTranscript();
```

Comment:
Could create small helper function which will create objects for you like transcript, student.